src/main/kotlin/net/axay/hglaborlobby/main/Manager.kt

```diff
@@ -33,14 +30,19 @@ class InternalMainClass : KSpigot() {
 
     override fun load() {
         INSTANCE = this
-
+        System.setProperty(
```

Comment:
funktioniert das nicht auch wenn es im init block ganz oben im database object ist?

Reply:
hatte ich eigentlich probiert gehabt ausserdem ist es schöner das beim load zu machen als in so einen init block

Reply:
wie du meinst aber generell ist es schon besser das da zu machen wo es hingehört und nicht in die main load function - egal jetzt aber erstmal